packages/sdk/src/configure/index.ts

```
@@ -1192,6 +1214,86 @@ const trimTrailingSlash = (url: string): string => {
 	return url.endsWith('/') ? url.slice(0, -1) : url;
 };
 
+// typeScriptOperationsResponseSchemas generates the response schemas for all TypeScript
+// operations at once, since it's several times faster than generating them one by one
+const typeScriptOperationsResponseSchemas = (operations: GraphQLOperation[]) => {
+	const functionTypeName = (op: GraphQLOperation) => `function_${op.Name}`;
+	const responseTypeName = (op: GraphQLOperation) => `${functionTypeName(op)}_Response`;
+
+	const programFile = 'typescript_schema_generator.ts';
+
+	let contents: string[] = ['import type { ExtractResponse } from "@wundergraph/sdk/operations";'];
+
+	for (const op of operations) {
+		const relativePath = `../operations/${op.PathName}`;
+		const name = functionTypeName(op);
+		contents.push(`import type ${name} from "${relativePath}";`);
+		contents.push(`export type ${responseTypeName(op)} = ExtractResponse<typeof ${name}>`);
+	}
+
+	const cachePath = path.join('cache', `ts.operationTypes.${objectHash(contents)}.json`);
+	if (fs.existsSync(cachePath)) {
+		try {
+			const cached = fs.readFileSync(cachePath, { encoding: 'utf-8' });
+			return JSON.parse(cached) as Record<string, JSONSchema>;
+		} catch {}
```

Empty catch?

---

That's right. If the cache loading fails (maybe the file ended up corrupted somehow), we'd rather fail silently and try to regenerate everything as if there was no cache entry.

---

@fiam put this explanation as a comment please